andela/casablanca-rc
# feature(Improved Image Gallery):

#### What does this PR do?
* add Landing page and images for products

#### Description of Task to be completed?
* create the template for the landing page
* add style for the homepage template
* create different product categories
* subscribe to the database to get the list of tags

#### How should this be manually tested?
* clone this repo
* cd into the project folder
* run `meteor npm install` to install project dependencies
* run `npm install ImageMagick`
* run reaction to start up the app
* the landing page should be loaded
* toggle the edit mode icon
* click on the edit symbol 
* add shop categories

#### What are the relevant pivotal tracker stories? (if applicable)
157032078

#### Screenshots

<img width="1440" alt="screen shot 2018-05-10 at 11 22 34 am" src="https://user-images.githubusercontent.com/29184825/39872691-b541849a-5460-11e8-8c46-5da6d8088000.png">
<img width="1440" alt="screen shot 2018-05-10 at 11 22 53 am" src="https://user-images.githubusercontent.com/29184825/39872865-35bd88ee-5461-11e8-87ab-670e56d0559c.png">
<img width="1440" alt="screen shot 2018-05-10 at 11 22 48 am" src="https://user-images.githubusercontent.com/29184825/39873195-260174c8-5462-11e8-871a-7c0f55e693ec.png">

<img width="1440" alt="screen shot 2018-05-10 at 1 40 27 pm" src="https://user-images.githubusercontent.com/29184825/39873028-bd3eaf00-5461-11e8-9cb3-e770f8e213d4.png">
